---
title: System.ServiceModel.AsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: dfc27ca7-b904-4f29-a5b6-dc702fa54a20
ms.openlocfilehash: 24cfab417e078e3d57f1b3f6fb75aeea1234afe2
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: MT
ms.contentlocale: fr-FR
ms.lasthandoff: 04/23/2019
ms.locfileid: "61752774"
---
# <a name="systemservicemodelasynccallbackthrewexception"></a>System.ServiceModel.AsyncCallbackThrewException
System.ServiceModel.AsyncCallbackThrewException  
  
## <a name="description"></a>Description  
 Un rappel asynchrone a levé une exception.  
  
## <a name="see-also"></a>Voir aussi

- [Suivi](../../../../../docs/framework/wcf/diagnostics/tracing/index.md)
- [Utilisation du suivi pour résoudre les problèmes posés par votre application](../../../../../docs/framework/wcf/diagnostics/tracing/using-tracing-to-troubleshoot-your-application.md)
- [Administration et diagnostics](../../../../../docs/framework/wcf/diagnostics/index.md)
